website/ember_osf_web/decorators.py

```diff
@@ -33,3 +33,8 @@ def wrapped(*args, **kwargs):
 def storage_i18n_flag_active():
     request.user = _get_current_user() or MockUser()
     return waffle.flag_is_active(request, 'storage_i18n')
+
+
+def storage_usage_flag_active():
+    request.user = _get_current_user() or MockUser()
+    return waffle.flag_is_active(request, 'storage_usage')
```

Comment:
Can you please move `'storage_usage'` to a constant in `osf/features.py` and use that here?